### DISCUSSION TITLE
Code Help: Showing gifts on desk

### DISCUSSION THREAD

**Discussion by WiiBii2** [Monika-After-Story/MonikaModDev#8081] [QA - code and content creation (technical)] [answered]
Hello again! I was looking to add some variation in Monika's responses and greetings. One idea I had was when the player came back from a certain brb, they would catch Monika drinking coffee and humming to herself. The problem is I do not know how to show said coffee on her desk. Can anyone point me in the right direction?

**Comment by lunulae** [comment]
If you start a brb during the time that Monika would normally brew coffee for herself (morning hours) you already might come back to find she's made a cup while you were gone.
I do like the idea of perhaps attaching some dialogue to instances like this, small talk breathes a lot more life into the game.
I'm trying to remember if we have a conditional for if she's having a particular food or drink, as that would be part of the setup.

**Comment by ThePotatoGuy** [comment, marked as answer]
Coffee is an ACS, which allows it to sit to be tied with Monika (and the desk). See [mas_acs_mug](https://github.com/Monika-After-Story/MonikaModDev/blob/master/Monika%20After%20Story/game/zz_spriteobjects.rpy#L1463) off of zz_spriteobjects. You can use the wear/remove ACS APIs to show/hide the coffee mug.
